src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
```

Comment:
Ensure that the font-weight specified here matches the actual weight of the 'Avenir-Book.ttf' font file. If the font file is not actually 300, it might cause rendering issues.